NOTICES

```diff
@@ -15609,7 +15248,7 @@ a Contributor.
 
   10.1. New Versions
 
-  Mozilla Foundation is the license steward. Except as provided in Section 10.3,
+  Mozilla Foundation is the license steward. Except as provided in Section��10.3,
```

Comment:
looks like it is still partially broken

Reply:
Yes, but now it also has real changes. I felt I could be picky about encoding issues when that was the only change, but ultimately I think having the correct versions is more important than not having a couple badly encoded characters. So in my estimation, worth merging despite the issue. I'll let you make the final call, though.

Reply:
yeah you’re right. I don’t really care about the encoding stuff.